fdbcli/fdbcli.actor.cpp

```diff
@@ -3656,7 +3657,30 @@ ACTOR Future<int> runCli(CLIOptions opt) {
 		[](std::string const& line, std::vector<std::string>& completions) {
 			fdbcli_comp_cmd(line, completions);
 		},
-		[](std::string const& line)->LineNoise::Hint {
+		[enabled=opt.cliHints](std::string const& line)->LineNoise::Hint {
+			if (enabled) {
```

Comment:
Since you are already fairly deeply nested it might read clearer to just have:
```cpp
if (!enabled) return LineNoise::Hint();
```

Reply:
Done.